internal/services/dynatrace/dynatrace_monitors_resource_test.go

```diff
@@ -0,0 +1,196 @@
+package dynatrace_test
```

Comment:
Can we add a Precheck for all these test cases, as they are expensive to test (the [price](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/dynatrace.dynatrace_portal_integration?tab=PlansAndPrice))? The Precheck can be a check on a specific environment variable, that will skip the test case when that variable is not specified explicitly.

Reply:
Sure. Will add.